### ISSUE TITLE
Sphax PureBD Patch crashes VoxelMap

### ISSUE THREAD

**Issue by soccerjoshj07** [xJon/The-1.12.2-Pack#421]
**Basic details**
1. Are you using a legitimate launcher and Minecraft account:
**yes**.
2. Are you using a computer with 8GB or more of RAM? ([How to check](https://www.wikihow.com/Check-Computer-RAM)):
**yes**.
3. Are you using Java 8, 64-bit? ([How to check](https://www.wikihow.com/Determine-Java-Version)):
**yes**.

**Information required**
1. Is the modpack crashing: **yes**
2. Crash/latest log *link* ([Where to find](https://github.com/xJon/The-1.12.2-Pack/blob/master/log_location.md#finding-modpack-installation-folder)): **https://paste.ubuntu.com/p/YfS8PMVfyS/**
3. Is Optifine installed or any other additional mods: **no**

**Describe the issue**

Perhaps this isn't the right place to post this, considering this is a resource pack, but I was using the resource pack listed in the post (https://bit.ly/The-1-12-2-Pack-Sphax) and receive an error that relates to voxelmap. I could disable voxelmap and the resource pack loads just fine, but otherwise the resource pack doesn't work for me and a friend who tried this out. I just have the 64x64 sphax resource pack enabled without the patch so I can play with voxelmap. 

Using version 1.12.2, downloaded fresh a few days ago.

**Comment by xJon** [comment]
`java.lang.OutOfMemoryError` -> you need to allocate more RAM to the modpack.
How much do you have on your computer?

**Comment by soccerjoshj07** [comment]
Thanks for the reply. I have 16gb of RAM on this machine. I was originally running with 5gb in the log file provided. 

I tried increasing to 8gb of ram, and it crashed again (same error, but 6x larger log file, if that's useful).

I increased to 12gb of ram and was able to open my world successfully. I'm using the 64x64 version of the resource pack. Is this expected behavior / does the patched resource pack really require so much RAM? I'm sure I could get by with less RAM if I used the 32x32 version?

Thanks
